goentri.com.nostra.json

```diff
@@ -0,0 +1,22 @@
+{
+  "providerId": "goentri.com",
+  "providerName": "Entri",
+  "serviceId": "nostra",
+  "serviceName": "Nostra",
+  "version": 1,
+  "logoUrl": "https://cdn.goentri.com/logo.svg",
+  "description": "Allows user to easily set up domain using Entri",
+  "syncPubKeyDomain": "goentri.com",
+  "syncRedirectDomain": "api.goentri.com, goentri.com, entri.com, app.entri.com,app.goentri.com",
+  "variableDescription": "mappingValue is the nostra's target domain",
+  "sharedProviderName": true,
+  "hostRequired": false,
```

Comment:
hostRequired shall be true if CNAME is at APEX

Reply:
Perfect @pawel-kow , set to true, thanks!

Reply:
let me know if there something left to fix here @pawel-kow . thanks a lot